components/Main/Workouts/WorkoutDetails/StyledWorkoutDetails.js

```diff
@@ -1,24 +1,62 @@
 import styled from "styled-components";
 
 export const StyledWorkoutDetails = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  li {
+    padding: 0rem 1rem;
+    list-style: none;
+    background-color: #f8f9fa;
+    color: black;
+    margin: 0.5rem 0.3rem;
+    border-radius: 0.2rem;
+    box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.26);
+    text-decoration: none;
+    .addContainer {
+      display: flex;
+      width: 100%;
+      justify-content: space-between;
+      height: 2.2rem;
+      align-items: center;
+    }
+    &:active {
+      transform: scale(97%);
+      transition: all 200ms ease;
+    }
+  }
+
   .addButton {
+    display: flex;
     height: 3.2rem;
     width: 100%;
     overflow: hidden;
     position: fixed;
     bottom: 0;
-    /* border-top-left-radius: 0.5rem;
-    border-top-right-radius: 0.5rem; */
+
     border-radius: 0.5rem;
     border: none;
     background: #99d98c;
     box-shadow: inset 0px 0px 0px 3px rgba(0, 0, 0, 0.1);
+    justify-content: center;
+    align-items: center;
+    color: black;
+    font-family: Lato-bold;
```

Comment:
diese Schriftart benutzt du überall oder? du könntest die sonst irgendwann noch in die Global Styles schreiben - wenn Zeit ist 🙂

Reply:
die reguläre font family hat keine bold. deswegen führe ich sie dort auf wo ich es fett haben will